### PR TITLE
Improvements 

### PR DESCRIPTION
**Pull request by paularah**
@prosazhin It'd be nice if you help vet this design wise.

### CODE REVIEW

**Review by prosazhin — APPROVED** [review]

File: src/components/pages/get-started/architecture/architecture.jsx [hunk 1]
```diff
@@ -3,21 +3,28 @@ import React from 'react';
...
 
-import diagram from './images/diagram.svg';
-
 const title = 'Architecture';
 const description =
-  'Cilium consists of an agent running on all cluster nodes and servers in your environment.  It provides networking, security, and observability to the workloads running on that node. Workloads can be containerized or running natively on the system.';
-
+  "Cilium comprises four key components: the Cilium agent, the Cilium client command line tool, the Cilium operator, and the Cilium CNI plugin. The agent, running on all cluster nodes, configures networking, load balancing, policies, and monitoring via Kubernetes or APIs that describe networking, service load-balancing, network policies, and visibility & monitoring requirements. The client tool, bundled with the agent, inspects and manages the local agent's status, offering direct access to eBPF maps. The operator centrally manages cluster tasks, handling them collectively rather than per node. The CNI plugin, invoked by Kubernetes during pod scheduling or termination, interacts with the node's Cilium API to configure necessary datapaths for networking, load balancing, and network policies.";
```
Comment (xmulligan) [inline src/components/pages/get-started/architecture/architecture.jsx:8]: did you pull this from somewhere?

File: src/components/pages/get-started/architecture/architecture.jsx [hunk 2]
```diff
@@ -3,21 +3,28 @@ import React from 'react';
...
         <p className="mt-5 md:text-lg">{description}</p>
       </div>
-      <div className="col-span-full mx-auto lg:col-span-7">
-        <img className="xl:ml-auto xl:max-w-[664px]" src={diagram} alt="Diagram" loading="lazy" />
+      <div className="flex flex-col justify-center it..ems-center lg:items-start lg:col-span-7">
+        <iframe
+          className="xl:max-w-[664px] w-full"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/LCQ89uBB7zE?si=udmn0u7mTWY_dbqK"
```
Comment (xmulligan) [inline src/components/pages/get-started/architecture/architecture.jsx:23]: not sure we should use an isovalent video. is there an OSS one from echo we could use?

Comment (paularah) [reply src/components/pages/get-started/architecture/architecture.jsx:23]: I thought about this too but I couldn't find a good OSS or echo video

File: src/components/pages/get-started/architecture/architecture.jsx (hunk 1 above)
Comment (paularah) [reply src/components/pages/get-started/architecture/architecture.jsx:8]: I wrote this. But it's basically a summary of the component overview section in the docs
https://docs.cilium.io/en/stable/overview/component-overview/

File: src/components/pages/get-started/hero/hero.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        </Heading>
+        <div
+          className="text-md with-link-primary mt-5 space-y-5 leading-relaxed md:text-lg md:leading-relaxed lg:max-w-[503px]"
+          dangerouslySetInnerHTML={{ __html: description }}
+        />
+      </div>
+      <div className="relative flex flex-1 justify-center pt-10">
+        <iframe
+          className="w-full"
+          src="https://www.youtube.com/embed/yjLJ2mib2zI?si=ry_mku6SshRhOIpb"
```
[the remark this answers is not in the document]
Comment (paularah) [reply src/components/pages/get-started/hero/hero.jsx:41]: Yes, same. Might be idea for a Echo episode or video reintroducing Cilium

[the remark this answers is not in the document]
Comment (xmulligan) [reply src/components/pages/get-started/hero/hero.jsx:41]: Ok, let's go with https://www.youtube.com/watch?v=80OYrzS1dCA for now and we can have a better video in the future

File: src/components/pages/get-started/architecture/architecture.jsx (hunk 2 above)
Comment (xmulligan) [reply src/components/pages/get-started/architecture/architecture.jsx:23]: Ok, let's use this one for now and update in the future when we have a better architecture one

File: src/components/pages/get-started/hero/hero.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (paularah) [reply src/components/pages/get-started/hero/hero.jsx:41]: we already have this^

image

**Review by xmulligan — APPROVED** [review]

**Pull request merged**